### ISSUE TITLE
AccessibilityInsights: Misplaced bounding rectangle for ComboBox items when drop down shows a scrollbar 

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#4323]
* .NET Core Version: 
6.0.100-alpha.1.20603.2

* Have you experienced this same bug with .NET Framework?: 
Yes, 
This is a regression issue, can repro in .Net 4.8, and cannot repro in .Net 4.7.2.

**Minimal repro:**
1. Create a Winforms .Net Core app with 3 ComboBox controls,  set their's DropDownHeight property to show scrollbar for comboBox list
Or open attached sample project and run Core app directly
[CoreTesting4.zip](https://github.com/dotnet/winforms/files/5640705/CoreTesting4.zip)
2. Use [Accessibility Insights](https://accessibilityinsights.io/en/downloads) tool to test ComboBox dropdown items

**Actual behavior:**
For some ComboBox list items, there is wrong bounding rectangle if scrollbar is not in minimal position(value 0, scrolled to top)
AccessibilityInsights:
![image](https://user-images.githubusercontent.com/26474449/101426857-b99da400-3938-11eb-9d29-03efb0f37bf1.png)
Inspect:
![image](https://user-images.githubusercontent.com/26474449/101427141-4cd6d980-3939-11eb-8b2c-65670551d4e9.png)

**Expected behavior:**
ComboBox items bounding rectangle for accessibility Insights are in correct position: in the list pane.
![image](https://user-images.githubusercontent.com/26474449/101124673-75f22400-3632-11eb-8c7d-3134f3ba5425.png)

**Comment by Amy-Li03** [comment]
Verified this issue on .NET core 3.1.12, the position of bounding rectangle for ComboBox items is correct when drop down shows a scrollbar.
![Inspect](https://user-images.githubusercontent.com/62929087/105683278-728e6100-5f2e-11eb-8455-44e5b1251129.gif)

After testing ScrollIntoView method, It does nothing if a tested item is visible, and move a tested item to the top of a tested invisible ComboBox item.

![Fixed](https://user-images.githubusercontent.com/62929087/105682388-5a6a1200-5f2d-11eb-90d5-efa6b0f82749.gif)

**Comment by Olina-Zhang** [comment]
Verified this issue in .Net 5.0.103-servicing.21073.8 build, this issue is fixed, the result is same as above @Amy-Li03 's.
